daemon-systemd.in

```diff
@@ -10,7 +10,7 @@ PIDFile=@lockfile@
 PermissionsStartOnly=true
 ExecStartPre=-/usr/bin/mkdir /var/run/naemon
 ExecStartPre=/usr/bin/chown -R naemon:naemon /var/run/naemon/
-ExecStartPre=@bindir@/naemon -vp @pkgconfdir@/naemon.cfg
+ExecStartPre=su - naemon -c "@bindir@/naemon -vp @pkgconfdir@/naemon.cfg"
```

Comment:
Thats only true for "ExecStart", the pre commands are run as root because we have set PermissionsStartOnly=true. So without the su, systemd trys to run the config check as root with is not a good idea and doesn't work anymore.